src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -141,6 +141,13 @@ public class SFSession extends SFBaseSession {
    */
   private int retryTimeout = 300;
 
+  /**
+   * Max timeout for external browser authentication in seconds
+   *
+   * <p>Default: 120
+   */
+  private int browserResponseTimeout = 120;
```

Comment:
do we need to keep default in two places? let's have a constant in this class and reuse it in SFLoginInput if necessary